.github/workflows/main.yml

```diff
@@ -20,9 +20,33 @@ jobs:
         run: make check PYTHON=python3
       - name: Rename
         run: cp mavlink-routerd mavlink-routerd-libc-x86-64
+      - name: package-deb
+        run: |
+          mkdir -p .debpkg/usr/bin
+          cp mavlink-routerd .debpkg/usr/bin/mavlink-routerd
+          mkdir -p .debpkg/DEBIAN
+          export VERSION=$([[ "${{ github.ref }}" == refs/tags/* ]] && echo $(echo ${{ github.ref }} | sed s#"refs/tags/v"##) || echo "0.0.0")
+          export PKGSIZE=$(du mavlink-routerd)
+          echo "Building mavlink-router version: ${VERSION}"
+          cat > .debpkg/DEBIAN/control << EOL
+          Package: mavlink-router
+          Version: ${VERSION}
+          Section: custom
+          Priority: optional
+          Architecture: amd64
+          Essential: no
+          Installed-Size: ${PKGSIZE}
```

Comment:
is this really needed?

Reply:
I assume architecture is definitely a needed field. The other fields, if I recall correctly, are also recommended by Debian.

Reply:
My comment was in the line just above, "Installed-Size". This should be calculated by the debian tooling, shouldn't it?

Reply:
It doesn't appear to be calculated from what I can gather. Omitting it and then building leads to apt reporting an installed size estimate of 0B.